.github/workflows/test.yml

```diff
@@ -41,14 +41,14 @@ jobs:
   docs:
     name: Docs
     runs-on: solang-ubuntu-latest
-    container: ubuntu:20.04
+    container: ubuntu:22.04
     steps:
       - name: Checkout sources
         uses: actions/checkout@v3
       - name: Install Python and git
         run: |
           apt-get update
-          apt-get install -y python3-pip git
+          apt-get install -y python3-pip git pkg-config libcairo-dev
```

Comment:
These changes are also in #1298. I think it is best the leave them only in one place.

Reply:
That was done to make the tests pass. Once one PR is merged, the rebase will remove the duplicate code

Reply:
> That was done to make the tests pass. Once one PR is merged, the rebase will remove the duplicate code

This is why these changes better suit in a separate PR.